mkt/site/monitors.py

```diff
@@ -240,7 +241,7 @@ def package_signer():
         msg = 'Error on package signing (%s): %s' % (destination, e)
         return msg, msg
     finally:
-        storage.delete(signed_path)
+        local_storage.delete(signed_path)
```

Comment:
So that part makes sense as `signed_path` is a local `tempfile`, but it's then used as the `dest` argument when calling `sign_app()`, which has this:
```
storage = private_storage if reviewer else public_storage
...
with open(tempname) as temp_f, storage.open(dest, 'w')
```

So the `signed_path` will be opened by `sign_app()` using remote storage. Not sure how to fix this. Maybe consider copying the file to remote (public) storage first ?